### DISCUSSION TITLE
Starknet-Foundry Developers Call

### DISCUSSION THREAD

**Discussion by amanusk** [foundry-rs/starknet-foundry#443] [Announcements]
## This is an event for a recurring weekly meeting.

Developers who want to ask questions about implementation, work on features, suggest improvements are welcome to join.

## Next Meeting Schedule:
Thursday, August 10th, 12PM UTC

Link: [zoom](https://starkware-co.zoom.us/j/87982010509)

**Comment by Arcticae** [comment]
## 📣 **Summary from Starknet-Foundry team for this week  (10.08)** 📣

### Upcoming in next release (16.08):
 ⁃ `#[should_panic]` attribute
 ⁃ `snforge_std` exported as a scarb package (syntax highlighting in vscode should work)
 ⁃ Custom `class_hash` for account creation in `sncast`

### In progress:
 ⁃ Precalculating contract address before deploying
 ⁃ Loading flat & json file data in tests
 ⁃ Bump of `cairo` to 2.1.0
 ⁃ `mock_call` cheatcode
 ⁃ `get_class_hash` cheatcode - first community made cheatcode🔥

### In design:
 ⁃ `expect_events` cheatcode
 ⁃ RPC State forking for tests
 ⁃ Deployment scripts - `deploy`, `declare`, `call` to testnet straight from Cairo code!

### Also:
 ⁃ Various bug fixes and refactoring

PS: A big thank you for all new contributions, the engagement from the community is amazing!
PS2: Make sure to check out `good-first-issue`s regularly, we keep adding them, so you guys can find an easy way to contribute! https://github.com/foundry-rs/starknet-foundry/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
